Sources/Tabs/TabManagerCoordinator.swift

```diff
@@ -136,6 +139,8 @@ extension TabManagerCoordinator: ParentCoordinator {
                     throw SecureStoreError.cantDeleteKey
                 }
                 #endif
+                // Reset the wallet service accessed before state to false
+                walletAvailabilityService.resetFeatureState()
                 try sessionManager.clearAllSessionData()
```

Comment:
For this, I've added the reset of the flag here - wasn't sure if it would be better suited in the `PersistentSessionManager` (where everything else is deleted) - let me know if better to move it there

Reply:
Are we able to subscribe `WalletAvailabilityService` to the `SessionBoundData` protocol?
Then we can tie it to the session manager delete method.

This should be managed by the session business logic rather than the coordinator (which should only be responsible for UI) 🙂

Reply:
Done, thank you 🙂